pkgs/applications/networking/termius/default.nix

```diff
@@ -12,12 +12,11 @@
 
 stdenv.mkDerivation rec {
   pname = "termius";
-  version = "7.15.1";
+  version = "7.16.0";
 
   src = fetchurl {
-    # Termius switched to using non-versioned downloads https://s3.amazonaws.com/termius.desktop.autoupdate/linux/Termius.deb
-    url = "https://web.archive.org/web/20210710174019/https://s3.amazonaws.com/termius.desktop.autoupdate/linux/Termius.deb";
-    sha256 = "16zc7ywz3hl1awkc4wk0rd94nsy55l98j2yzfdxcjiixky4gk8wn";
+    url = "https://deb.termius.com/pool/main/t/termius-app/termius-app_${version}_amd64.deb";
```

Comment:
I'd add a comment about using `wget -qO- https://deb.termius.com/dists/squeeze/main/binary-amd64/Packages` to get the latest link.

Reply:
```suggestion
    # find the latest version by
    # curl https://deb.termius.com/dists/squeeze/main/binary-amd64/Packages
    url = "https://deb.termius.com/pool/main/t/termius-app/termius-app_${version}_amd64.deb";
```